Library/Numbers+Time.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+extension TimeInterval {
+    var day: TimeInterval {
+        return self.hour * 24
+    }
+
+    var hour: TimeInterval {
+        return self.minute * 60.0
+    }
+
+    var minute: TimeInterval {
+        return self * 60.0
```

Comment:
why make all these methods on `timeInterval` and `int` instead of using date components?

Reply:
Not sure how that would work.

Reply:
Ah, i see where you're using them at the call site, nevermind.